code/modules/mob_spawn/ghost_roles/mining_roles.dm

```diff
@@ -294,7 +294,7 @@
 
 /obj/effect/mob_spawn/ghost_role/human/lavaland_syndicate/special(mob/living/new_spawn)
 	. = ..()
-	new_spawn.grant_language(/datum/language/codespeak, TRUE, TRUE, LANGUAGE_MIND)
+	new_spawn.grant_language(/datum/language/codespeak, TRUE, TRUE, LANGUAGE_SPAWNER) // SKYRAT EDIT CHANGE - ghost roles require LANGUAGE_SPAWNER to prevent loading prefs from overwriting languages granted in special()
```

Comment:
Add the original line after a `ORIGINAL: originalCodeHere` at the end of the comment please :)